### ISSUE TITLE
Delegate vote 'number' as string??

### ISSUE THREAD

**Issue by tomstuart123** [decentdao/decent-interface#1830]
![Image](https://github.com/decentdao/fractal-interface/assets/31554133/913ebe3c-8622-4461-a06c-625448d28cd9)

See above. When I delegated a token to another (1000).. but then received more token  from someone else (100), the tokens seemed to bolt on to create 100100... instead of the desired 1100

My guess is we handle them as strings post calculation

**Comment by Da-Colon** [comment]
Weird...this shouldn't happen there isn't any 'adding' going on. We re-request the `balanceOf` when an event happens

**Comment by adamgall** [comment]
@tomstuart123 adding `re-skin` label

**Comment by Da-Colon** [comment]
Will need to ssee this one live I think to see whats going on, can't replicate locally.

**Comment by tomstuart123** [comment]
lets tackle the other stuff I'm facing first. We can leave this in backlog as a nice to have.

Moving to p2 for now as its an edge case of multiple delegations

**Comment by adamgall** [comment]
@tomstuart123 can you point to the transactions onchain that led to this scenario in Decent UI?

Ideally I'd like to see the transactions which:
- minted tokens to 0xf26d 
- 0xf26d delegated tokens (to self or anyone else)
- 0xf26d received tokens (from someone else)
- 0xf26d received delegation (from someone else) (did this happen?)

**Comment by tomstuart123** [comment]
will do. This might take a while so - if okay -  I'll wait to do this until we make our way through the other p1s. want to unblock those first and get past launch

**Comment by adamgall** [comment]
@tomstuart123 the reason i'm asking you is because i couldn't find those transactions onchain myself.

what you're describing is pretty strange behavior, so i just wanted to see what the data for this issue looks like onchain. it's not clear to me from the original description which addresses were involved in this issue, or the order that steps were taken.

so really, i'm questioning whether the issue is with our app, or something else. the data onchain don't lie

**Comment by tomstuart123** [comment]
makes sense. When I have time I'll look at it and share more details of the onchain stuff

**Comment by adamgall** [comment]
@tomstuart123 i'm going to close this, please re-open if you're able to reproduce
